test/testLiveAsTemplate.js

```diff
@@ -124,6 +124,10 @@ describe('CLI interface, oghliner as a template', function() {
     .then(function() {
       assert.doesNotThrow(fse.statSync.bind(fse, 'index.html'));
       assert.doesNotThrow(fse.statSync.bind(fse, 'offline-worker.js'));
+    })
+    .catch(function(err) {
+      console.log('Error: ' + err);
+      throw err;
```

Comment:
Basically, the getTokenId promise was never resolved (because it kept trying with higher and higher page numbers), so the mocha test was never ending (the promise in *after* was never resolved).